README.md

```diff
@@ -0,0 +1,26 @@
+# Test Doubles Exercise 
+
+Test doubles exercise as part of the MCR Codes Fundamentals module. It is done by Chris Peakin, Dan Hope, July Moss and Stuart Green as a group progamming practice. 
```

Comment:
Nice to see multiple contributors! I can't see Dans name anywhere in the attributions though, was he helping but didnt submit any actual code?